src/styles/app.css

```diff
@@ -1,6 +1,6 @@
 /* Base Application Styles */
 
-.app {
+main {
   color: #666;
```

Comment:
What about using `#app` instead? Would rather stick to traditonal html elements like `<div>` to not throw another curve at them.